backend/Dockerfile

```
@@ -1,10 +1,25 @@
-FROM tiangolo/uvicorn-gunicorn-fastapi:python3.9-slim
+ARG APP_ENVIRONMENT=dev
+
+
+FROM python:3.9-slim as base
```

I think there's a problem with the 3.9 image and M1 drivers

```suggestion
FROM --platform=linux/amd64 python:3.9-slim as base
```

This seems to fix it...
Explanation here https://stackoverflow.com/questions/65612411/forcing-docker-to-use-linux-amd64-platform-by-default-on-macos/69636473#69636473

---

I pushed this change